src/main/java/com/puppycrawl/tools/checkstyle/checks/header/RegexpHeaderCheck.java

```diff
@@ -151,9 +167,15 @@ private boolean isMultiLine(int lineNo) {
     @Override
     protected void postProcessHeaderLines() {
         final List<String> headerLines = getHeaderLines();
-        for (String line : headerLines) {
+        final Pattern blankLine = Pattern.compile(EMPTY_LINE_PATTERN);
+        for (final String line : headerLines) {
```

Comment:
please remove this `final` it pollute code and we have special Check for this that should make change of value impossible.

Reply:
Done